src/OpenGL.cpp

```diff
@@ -1327,6 +1327,10 @@ void OGLRender::_initExtensions()
 	m_bImageTexture = false;
 #endif
 	LOG(LOG_VERBOSE, "ImageTexture support: %s\n", m_bImageTexture ? "yes" : "no");
+	if (m_bImageTexture == false)
+	{
+		LOG(LOG_WARNING, "Image Textures not supported. Without OpenGL >= 4.3 or GLES >= 3.1, depth buffer emulation will not work");
```

Comment:
BAR fog was initially implemented with Image textures, but then I remade it and now it uses standard FBO depth texture. However it still uses Image texture for depth LookUpTable. I checked BAR with m_bImageTexture=false, it really turns black on levels with fog. 

So, "N64 depth compare and depth based fog will not work without  Image Textures".